hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/TestFileCreation.java

```diff
@@ -282,7 +282,7 @@ public void testServerDefaultsWithMinimalCaching() throws Exception  {
           // do nothing;
           return false;
         }
-      }, 1, 3000);
+      }, 1, 6000);
```

Comment:
The comment `Wait for 3 seconds` above also needs to be changed.

Reply:
Thanks for your help review, I will fix it.